<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LarvaResForm</class>
 <widget class="QWidget" name="LarvaResForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Larve</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widgetChanged" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Larvenposition wurde manuell geändert</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButtonApplyChanges">
              <property name="text">
               <string>Änderungen übernehmen</string>
              </property>
              <property name="icon">
               <iconset resource="qttestvs10.qrc">
                <normaloff>:/QTTestVs10/rect2985.png</normaloff>:/QTTestVs10/rect2985.png</iconset>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReset">
              <property name="text">
               <string>Zurücksetzen</string>
              </property>
              <property name="icon">
               <iconset resource="qttestvs10.qrc">
                <normaloff>:/QTTestVs10/exit.png</normaloff>:/QTTestVs10/exit.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonColorDialog">
          <property name="text">
           <string>Farbe:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelColorDialog">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBoxDisplayLarva">
          <property name="text">
           <string>Larve Anzeigen</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushFindLarva">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Szene zu dieser Larve scrollen</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="qttestvs10.qrc">
            <normaloff>:/QTTestVs10/pathfinder.png</normaloff>:/QTTestVs10/pathfinder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLabel" name="labelDist2Origin">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Entfernung zum Ursprung:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Interpoliertes Modell:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelAccDistance">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Gesamtfortschritt:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelInterpolated">
          <property name="text">
           <string>Nein</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonPrevCollision">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonNextCollision">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBoxDist2Origin">
          <property name="text">
           <string>Anzeigen</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBoxPath">
          <property name="text">
           <string>Pfad Anzeigen</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Zeit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderTime">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxTime"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Radius an Stelle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxRadiusPosition"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRadius">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string> Pixel</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushInvert">
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Modell invertieren</string>
        </property>
        <property name="icon">
         <iconset resource="qttestvs10.qrc">
          <normaloff>:/QTTestVs10/invert.png</normaloff>:/QTTestVs10/invert.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushUnite">
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Mit einem anderem Modell vereinigen</string>
        </property>
        <property name="icon">
         <iconset resource="qttestvs10.qrc">
          <normaloff>:/QTTestVs10/unite.png</normaloff>:/QTTestVs10/unite.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushDivide">
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Nach aktuellem Zeitschritt in neues Modell übertragen</string>
        </property>
        <property name="icon">
         <iconset resource="qttestvs10.qrc">
          <normaloff>:/QTTestVs10/split.png</normaloff>:/QTTestVs10/split.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Zeitschritt-Duplikat einfügen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushCopy2Prev">
          <property name="text">
           <string>davor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushCopy2Succ">
          <property name="text">
           <string>danach</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushDeleteCurrent">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Modell im aktuellen Zeitschritt löschen</string>
        </property>
        <property name="icon">
         <iconset resource="qttestvs10.qrc">
          <normaloff>:/QTTestVs10/exit.png</normaloff>:/QTTestVs10/exit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushDeleteLarva">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Larve aus Ergebnissen entfernen</string>
        </property>
        <property name="icon">
         <iconset resource="qttestvs10.qrc">
          <normaloff>:/QTTestVs10/exit.png</normaloff>:/QTTestVs10/exit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qttestvs10.qrc"/>
 </resources>
 <connections/>
</ui>
